doc/doxygen/stylesheet.css

```diff
@@ -26,3 +26,11 @@ div.doxygen-generated-exception-message {
     padding: 5px;
     font-size: 95%;
 }
+
+div.image img[src="fe_enriched_1d.png"] {
+    width:500px;
```

Comment:
here is what they say https://www.star.bnl.gov/public/comp/sofi/doxygen/commands.html

> The fourth argument is also optional and can be used to specify the width or height of the image. This is only useful for $\mbox{\LaTeX}$ output (i.e. format=latex). The sizeindication can be either width or height. The size should be a valid size specifier in $\mbox{\LaTeX}$ (for example 10cm or 6in or a symbolic width like \textwidth).